### ISSUE TITLE
Laravel 9 - TailwindCSS 3 - change theme does not  work

### ISSUE THREAD

**Issue by notwarp** [GeoffSelby/tailwind-highlightjs#3]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

used your package in laravel 9 but i cannot change the theme, i've changed the way you get the highlight.js's styles path

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/tailwind-highlightjs/src/utils.js b/node_modules/tailwind-highlightjs/src/utils.js
index bd21658..d8972fc 100644
--- a/node_modules/tailwind-highlightjs/src/utils.js
+++ b/node_modules/tailwind-highlightjs/src/utils.js
@@ -74,8 +74,8 @@ function isThemeOfficial(theme) {
 }
 
 function getOfficialTheme(theme) {
-  const themePath = path.resolve(
-    `${__dirname}/../node_modules/highlight.js/styles/${theme}.css`,
+  const themePath = path.join(
+    __dirname, "../..", `highlight.js/styles/${theme}.css`,
   );
   const themeContents = fs.readFileSync(themePath, 'utf8');
 
```

**Comment by renatoxm** [comment]
yes, that fixed the problem! Thanks @notwarp

**Comment by notwarp** [comment]
we should ask the maintainer to merge the branch for permanently have a fix

**Comment by GeoffSelby** [comment]
Sorry I haven't had a chance to look at this yet. It's on the list for this afternoon now. Thanks for being patient.

**Comment by GeoffSelby** [comment]
Fixed in v2.0.1

**Comment by notwarp** [comment]
Thanks @GeoffSelby
